src/+datools/+statistical/+ensemble/EnF.m

```diff
@@ -75,6 +79,15 @@
         end
 
         function forecast(obj)
+            % Description:
```

Comment:
Description here is redundant

Reply:
And again, follow the standard

`unction doThis(obj)
        % doThis  Do this thing
        %   Here is some help text for the doThis method.
        %
        %   See also DOTHAT.
        
        disp(obj)
        end        `